### DISCUSSION TITLE
DSL to SLOB Multimedia is not added

### DISCUSSION THREAD

**Discussion by gep-sher** [itkach/aard2-android#183] [Q&A]
Multimedia ( .jpg ) is not added.
Command:
(myenv) ..@.. convert % pyglossary file.dsl file.slob 

After conversion and when viewing the SLOB file, the images are not visible. Text format displayed correctly.
In DSL files:
• I use marks [s] [/s] - to mark a multimedia section (ABBYYLingvoDSL) and paths to the folder with images:
[s]name_folder/name_imag.jpg[/s]

All content ( folder_name.dsl.files , file.dsl , file_abrv.dsl , etc.) is located in the same folder, from where I issue the conversion command (in this case, the folder is named convert , see the code at the beginning).

What am I doing? Please help

**Comment by itkach** [comment]
This is a question about pyglossary, try asking in https://github.com/ilius/pyglossary
